notebooks/official/prediction/get_started_with_nvidia_triton_serving.ipynb

```diff
@@ -0,0 +1,1659 @@
+{
```

Comment:
<p>Update links to the notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2444/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>